app/src/test/java/com/distillery/interview/ui/CurrentWeatherViewModelTests.kt

```diff
@@ -56,8 +55,8 @@ class CurrentWeatherViewModelTests {
     @Test
     fun getCurrentWeather_returnErrorResponse() = runBlockingTest {
         //GIVEN
-        val mockError = Result.Error()
-        whenever(weatherRepository.getCurrentWeather()).thenReturn(mockError)
+        val mockError = RuntimeException()
```

Comment:
We can simulate an `IOException` since is more like to happen than a `RuntimeException`.

Reply:
now with runCatching we are not checking any exceptions in our methods, so they would all always be of type unchecked exceptions or runtime